### ISSUE TITLE
Address miscellaneous TODOs

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/number-suite-common#29]
This issue is a catch-all for miscellaneous TODOs that did not have an associated GitHub issue.  This issue covers the number-play, number-compare, number-suite, and counting-common repos.

These TODOs should be triaged for 1.0, identifying which ones need to be addressed.   If any of them require significant work, please split out into a new issue.

Assigning to @chrisklus to triage.

**Comment by pixelzoom** [comment]
Move to https://github.com/phetsims/number-suite-common/issues/29, closing.

**Comment by chrisklus** [comment]
Adding blocks-sim-publication label to indicate this should be completed before starting Number Compare and Number Play 1.0/1 dev testing.

**Comment by pixelzoom** [comment]
I took a quick look at all remaining TODOs.  I don't see any that seem essential to address before publishing Number Play or Number Compare (https://github.com/phetsims/number-compare/issues/9) -- i.e. nothing that impacts UX or crashes the sim.  @chrisklus you should definitely take a look too.   But I'd be OK with changing this issue from "blocks-sim-publication" to "deferred".

**Comment by zepumph** [comment]
I wasn't able to preserve git history for LocaleSwitch, here is the last version when it was still over in number-suite-common. https://github.com/phetsims/number-suite-common/blob/6fd98ef94e1cbc1320d35721983651af37985ac3/js/common/view/LocaleSwitch.ts

**Comment by zepumph** [comment]
Handled a good amount above, we will keep working on them! 11 left.

**Comment by zepumph** [comment]
- [x] While factoring out a button length, I was curious if @chrisklus could comment on if this value (20.8) was related to the resetAllButton constant in scenery-phet constants?

https://github.com/phetsims/number-compare/blob/78ed243599e0bd0a4af94d633663bec0a0bf20a9/js/compare/view/BlockValuesNode.ts#L17

**Comment by zepumph** [comment]
All TODO issues have been moved to side issues or handled in above commits. Closing
